From 4f59643043d7298da539615d64002be1ab2728e1 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Mon, 5 Oct 2026 14:41:09 +0000
Subject: [PATCH] Refresh the guest package lock

alsa-card-profiles 1:1.6.8-1 -> 1:1.6.9-1
aom 3.15.0-1 -> 3.15.1-1
appstream 1.2.0-1 -> 1.2.1-1
aquamarine 0.15.0-2 -> 0.15.1-1
at-spi2-core 2.60.6-1 -> 2.60.7-1
automake 1.18.1-1 -> 1.19-1
bash 5.3.15-1 -> 5.3.20-1
bat 0.26.1-2 -> 0.26.1-3
bubblewrap 0.12.0-1 -> 0.13.0-1
ca-certificates-mozilla 3.128-1 -> 3.130-1
cairo 1.18.4-1 -> 1.18.6-1
chromium 152.0.7977.82-1 -> 153.0.8010.52-1
clang 22.1.8-1 -> 23.1.1-1
compiler-rt 22.1.8-1 -> 23.1.1-1
coreutils 9.11-2 -> 9.12-2
device-mapper 2.03.42-1 -> 2.03.43-2
dua-cli 2.44.0-1 -> 2.45.1-1
expat 2.8.4-1 -> 2.8.5-1
fastfetch 2.68.1-1 -> 2.69.0-1
fcitx5 5.1.22-1 -> 5.1.23-1
fcitx5-qt 5.1.15-1 -> 5.1.16-1
ffmpeg 2:9.0.1-4 -> 2:9.0.2-2
fribidi 1.0.16-2 -> 1.0.17-1
fuse-common 3.18.2-1 -> 3.18.3-1
fuse3 3.18.2-1 -> 3.18.3-1
fzf 0.74.3-1 -> 0.74.4-1
gcr-4 4.4.0.1-1 -> 4.4.1-1
gdb 17.2-1 -> 18.1-1
gdb-common 17.2-1 -> 18.1-1
git 2.55.0-1 -> 2.56.0-1
glibc 2.44+r24+g16be1518495f-1 -> 2.44+r50+g1848099f063e-1
glslang 1:1.4.357.0-1 -> 1:1.4.363.0-1
gpm 1.20.7.r38.ge82d1a6-6 -> 1.20.7.r38.ge82d1a6-7
gpu-screen-recorder 6.1.0-1 -> 6.1.3-1
gst-plugins-bad-libs 1.28.7-1 -> 1.28.7-2
gst-plugins-base-libs 1.28.7-1 -> 1.28.7-2
gstreamer 1.28.7-1 -> 1.28.7-2
gtk-update-icon-cache 1:4.22.4-1 -> 1:4.22.5-1
gtk4 1:4.22.4-1 -> 1:4.22.5-1
gum 2.0.0-1 -> 2.0.2-1
gvfs 1.60.2-4 -> 1.60.3-3
gzip 1.14-2 -> 1.15-1
harfbuzz 14.4.0-1 -> 14.5.1-1
hyprland 0.56.2-2 -> 0.56.2-4
hyprland-guiutils 0.2.2-3 -> 0.2.2-4
hyprtoolkit 0.5.4-6 -> 0.6.0-1
imagemagick 7.1.2.31-1 -> 7.1.2.32-1
iniparser 4.2.6-2 -> 4.3.2-1
inotify-tools 4.25.9.0-1 -> 4.26.262-1
jemalloc 1:5.3.1-3 -> 1:5.4.0-1
leancrypto 1.9.0-1 -> 1.9.1-1
less 1:704-1 -> 1:710-1
libadwaita 1:1.9.3-1 -> 1:1.9.4-1
libcap-ng 0.9.5-1 -> 0.9.6-1
libde265 1.1.2-1 -> 1.1.3-1
libdwarf 1:2.3.2-1 -> 1:2.3.3-1
libgcrypt 1.12.3-1 -> 1.12.4-1
libgsf 1.14.58-1 -> 1.14.60-1
libheif 1.23.4-1 -> 1.23.5-1
libinput 1.31.3-1 -> 1.32.0-1
libldap 2.6.13-1 -> 2.7.1-1
libmd 1.2.0-1 -> 1.3.0-1
libnm 1.58.1-1 -> 1.58.1-2
libpcap 1.10.7-1 -> 1.11.0-1
libpipewire 1:1.6.8-1 -> 1:1.6.9-1
libplacebo 7.360.1-6 -> 7.360.1-7
libpng 1.6.58-2 -> 1.6.59-1
libportal 0.10.0-1 -> 0.11.0-1
libportal-gtk4 0.10.0-1 -> 0.11.0-1
librsvg 2:2.62.3-1 -> 2:2.62.4-1
libsecret 0.21.7-1 -> 0.21.8.2-1
libtirpc 1.3.7-1 -> 1.3.8-1
libutf8proc 2.11.3-1 -> 2.12.0-1
libvips 8.18.6-1 -> 8.18.7-1
libwireplumber 0.5.17-1 -> 0.5.18-1
linux 7.2.4.arch1-2 -> 7.2.8.arch1-2
llvm-libs 22.1.8-2 -> 23.1.1-1
md4c 0.5.3-1 -> 0.6.0-2
mesa 1:26.2.2-1 -> 1:26.2.4-1
mkinitcpio 41.1-1 -> 42.2-1
mujs 1.3.9-1 -> 1.3.10-1
networkmanager 1.58.1-1 -> 1.58.1-2
noto-fonts 1:2026.09.01-1 -> 1:2026.10.01-1
nss 3.128-1 -> 3.130-1
openexr 3.4.15-1 -> 3.5.2-1
openjph 0.31.0-1 -> 0.32.0-1
openssl 3.6.4-1 -> 3.6.5-1
orc 0.4.43-1 -> 0.4.44-1
pam 1.7.2-2 -> 1.7.3-1
parted 3.7-1 -> 3.8-1
pcre2 10.48-1 -> 10.49-1
pcsclite 2.5.1-1 -> 2.5.2-1
pipewire 1:1.6.8-1 -> 1:1.6.9-1
pipewire-alsa 1:1.6.8-1 -> 1:1.6.9-1
pipewire-audio 1:1.6.8-1 -> 1:1.6.9-1
pipewire-pulse 1:1.6.8-1 -> 1:1.6.9-1
python-dbus 1.4.0-2 -> 1.5.0-1
qt6-declarative 6.11.2-1 -> 6.11.2-2
readline 8.3.003-1 -> 8.3.6-1
sdl2-compat 2.32.72-1 -> 2.32.74-1
sdl3 3.4.16-1 -> 3.4.18-1
shaderc 2026.3-1 -> 2026.4-1
spirv-tools 1:1.4.357.0-1 -> 1:1.4.363.0-1
systemd 261.2-1 -> 262-1
systemd-libs 261.2-1 -> 262-1
systemd-resolvconf 261.2-1 -> 262-1
systemd-sysvcompat 261.2-1 -> 262-1
tzdata 2026c-1 -> 2026e-1
upower 1.91.3-1 -> 1.91.5-1
util-linux 2.42.3-1 -> 2.42.4-1
util-linux-libs 2.42.3-1 -> 2.42.4-1
uwsm 0.26.7-1 -> 0.27.0-1
vapoursynth 79-1 -> 80-1
vmaf 3.2.0-1 -> 3.2.1-1
vulkan-icd-loader 1.4.357.0-1 -> 1.4.363.0-1
vulkan-mesa-implicit-layers 1:26.2.2-1 -> 1:26.2.4-1
vulkan-swrast 1:26.2.2-1 -> 1:26.2.4-1
vulkan-virtio 1:26.2.2-1 -> 1:26.2.4-1
wireplumber 0.5.17-1 -> 0.5.18-1
xdg-terminal-exec 0.14.3-1 -> 0.14.3-2
xxhash 0.8.3-1 -> 0.8.4-1
xz 5.8.3-1 -> 5.8.4-1
yyjson 0.12.0-1 -> 0.13.0-1
numactl 2.0.19-1 (new)
hyprwayland-scanner (removed)
---
 guest/packages.lock.json | 248 +++++++++++++++++++--------------------
 1 file changed, 124 insertions(+), 124 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 1173597..641ac73 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -7,42 +7,42 @@
     "adwaita-fonts": "51.0-2",
     "adwaita-icon-theme": "50.0-1",
     "adwaita-icon-theme-legacy": "46.2-3",
-    "alsa-card-profiles": "1:1.6.8-1",
+    "alsa-card-profiles": "1:1.6.9-1",
     "alsa-lib": "1.2.16.1-1",
     "alsa-topology-conf": "1.2.5.1-4",
     "alsa-ucm-conf": "1.2.16.1-1",
-    "aom": "3.15.0-1",
-    "appstream": "1.2.0-1",
-    "aquamarine": "0.15.0-2",
+    "aom": "3.15.1-1",
+    "appstream": "1.2.1-1",
+    "aquamarine": "0.15.1-1",
     "archlinux-keyring": "20260909-1",
-    "at-spi2-core": "2.60.6-1",
+    "at-spi2-core": "2.60.7-1",
     "attr": "2.6.0-1",
     "audit": "4.2.1-1",
     "autoconf": "2.73-1",
-    "automake": "1.18.1-1",
+    "automake": "1.19-1",
     "avahi": "1:0.9rc5-1",
     "base": "3-3",
     "base-devel": "1-2",
-    "bash": "5.3.15-1",
+    "bash": "5.3.20-1",
     "bash-completion": "2.18.0-1",
-    "bat": "0.26.1-2",
+    "bat": "0.26.1-3",
     "binutils": "2.47-4",
     "bison": "3.8.2-8",
     "bluez-libs": "5.87-2",
     "boost-libs": "1.92.0-1",
     "brotli": "1.2.0-1",
     "btop": "1.4.7-1",
-    "bubblewrap": "0.12.0-1",
+    "bubblewrap": "0.13.0-1",
     "bzip2": "1.0.8-6",
     "ca-certificates": "20240618-1",
-    "ca-certificates-mozilla": "3.128-1",
+    "ca-certificates-mozilla": "3.130-1",
     "ca-certificates-utils": "20240618-1",
-    "cairo": "1.18.4-1",
+    "cairo": "1.18.6-1",
     "cfitsio": "1:4.7.0-1",
-    "chromium": "152.0.7977.82-1",
-    "clang": "22.1.8-1",
-    "compiler-rt": "22.1.8-1",
-    "coreutils": "9.11-2",
+    "chromium": "153.0.8010.52-1",
+    "clang": "23.1.1-1",
+    "compiler-rt": "23.1.1-1",
+    "coreutils": "9.12-2",
     "cpio": "2.15-3",
     "cpptrace": "1.0.4-2",
     "cryptsetup": "2.8.8-1",
@@ -57,26 +57,26 @@
     "debugedit": "5.3-2",
     "default-cursors": "3-1",
     "desktop-file-utils": "0.28-1",
-    "device-mapper": "2.03.42-1",
+    "device-mapper": "2.03.43-2",
     "diffutils": "3.12-2",
     "double-conversion": "3.4.0-1",
-    "dua-cli": "2.44.0-1",
+    "dua-cli": "2.45.1-1",
     "duktape": "2.7.0-7",
     "e2fsprogs": "1.47.4-1",
     "elfutils": "0.196-1",
     "enchant": "2.8.21-1",
     "exempi": "2.6.6-3",
     "exiv2": "0.28.9-1",
-    "expat": "2.8.4-1",
+    "expat": "2.8.5-1",
     "eza": "0.23.5-2",
     "fakeroot": "1:1.37.2-3",
-    "fastfetch": "2.68.1-1",
+    "fastfetch": "2.69.0-1",
     "fcft": "3.3.3-1",
-    "fcitx5": "5.1.22-1",
+    "fcitx5": "5.1.23-1",
     "fcitx5-gtk": "5.1.7-1",
-    "fcitx5-qt": "5.1.15-1",
+    "fcitx5-qt": "5.1.16-1",
     "fd": "10.5.0-3",
-    "ffmpeg": "2:9.0.1-4",
+    "ffmpeg": "2:9.0.2-2",
     "fftw": "3.3.11-1",
     "file": "5.48-1",
     "filesystem": "2025.10.12-1",
@@ -86,29 +86,29 @@
     "fontconfig": "2:2.18.3-2",
     "foot": "1.28.0-2",
     "freetype2": "2.14.3-1",
-    "fribidi": "1.0.16-2",
-    "fuse-common": "3.18.2-1",
-    "fuse3": "3.18.2-1",
-    "fzf": "0.74.3-1",
+    "fribidi": "1.0.17-1",
+    "fuse-common": "3.18.3-1",
+    "fuse3": "3.18.3-1",
+    "fzf": "0.74.4-1",
     "gawk": "5.4.1-1",
     "gc": "8.2.12-1",
     "gcc": "16.2.1+r23+gd564253eb6c8-1",
     "gcc-libs": "16.2.1+r23+gd564253eb6c8-1",
     "gcr": "3.41.2-2",
-    "gcr-4": "4.4.0.1-1",
-    "gdb": "17.2-1",
-    "gdb-common": "17.2-1",
+    "gcr-4": "4.4.1-1",
+    "gdb": "18.1-1",
+    "gdb-common": "18.1-1",
     "gdbm": "1.26-2",
     "gdk-pixbuf2": "2.44.7-1",
     "gettext": "1.0-2",
     "gexiv2": "0.16.2-2",
     "gexiv2-common": "0.16.2-2",
     "giflib": "6.1.3-2",
-    "git": "2.55.0-1",
+    "git": "2.56.0-1",
     "glib-networking": "1:2.80.1-1",
     "glib2": "2.88.3-1",
-    "glibc": "2.44+r24+g16be1518495f-1",
-    "glslang": "1:1.4.357.0-1",
+    "glibc": "2.44+r50+g1848099f063e-1",
+    "glslang": "1:1.4.363.0-1",
     "glu": "9.0.3-3",
     "glycin": "2.1.5-2",
     "glycin-gtk4": "2.1.5-2",
@@ -124,8 +124,8 @@
     "gperftools": "2.18.1-1",
     "gpgme": "2.2.0-1",
     "gpgmepp": "2.2.0-1",
-    "gpm": "1.20.7.r38.ge82d1a6-6",
-    "gpu-screen-recorder": "6.1.0-1",
+    "gpm": "1.20.7.r38.ge82d1a6-7",
+    "gpu-screen-recorder": "6.1.3-1",
     "graphene": "1.10.8-2",
     "graphite": "1:1.3.15-1",
     "grep": "3.12-2",
@@ -135,22 +135,22 @@
     "gsettings-system-schemas": "50.1-1",
     "gsm": "1.0.24-1",
     "gssdp": "1.6.6-1",
-    "gst-plugins-bad-libs": "1.28.7-1",
-    "gst-plugins-base-libs": "1.28.7-1",
-    "gstreamer": "1.28.7-1",
+    "gst-plugins-bad-libs": "1.28.7-2",
+    "gst-plugins-base-libs": "1.28.7-2",
+    "gstreamer": "1.28.7-2",
     "gtest": "1.18.0-1",
-    "gtk-update-icon-cache": "1:4.22.4-1",
+    "gtk-update-icon-cache": "1:4.22.5-1",
     "gtk3": "1:3.24.52-1",
-    "gtk4": "1:4.22.4-1",
+    "gtk4": "1:4.22.5-1",
     "gtk4-layer-shell": "1.3.0-1",
     "guile": "3.0.11-1",
-    "gum": "2.0.0-1",
+    "gum": "2.0.2-1",
     "gupnp": "1:1.6.10-1",
     "gupnp-dlna": "0.12.0-5",
     "gupnp-igd": "1.6.0-2",
-    "gvfs": "1.60.2-4",
-    "gzip": "1.14-2",
-    "harfbuzz": "14.4.0-1",
+    "gvfs": "1.60.3-3",
+    "gzip": "1.15-1",
+    "harfbuzz": "14.5.1-1",
     "hdf5": "2.2.0-1",
     "herdr": "0.8.2-1",
     "hicolor-icon-theme": "0.18-1",
@@ -161,23 +161,22 @@
     "hyprcursor": "0.1.13-7",
     "hyprgraphics": "0.5.1-4",
     "hypridle": "0.1.8-2",
-    "hyprland": "0.56.2-2",
-    "hyprland-guiutils": "0.2.2-3",
+    "hyprland": "0.56.2-4",
+    "hyprland-guiutils": "0.2.2-4",
     "hyprland-preview-share-picker": "0.2.1-1",
     "hyprlang": "0.6.8-5",
     "hyprpicker": "0.4.7-4",
     "hyprsunset": "0.4.0-3",
-    "hyprtoolkit": "0.5.4-6",
+    "hyprtoolkit": "0.6.0-1",
     "hyprutils": "0.14.2-1",
-    "hyprwayland-scanner": "0.4.6-1",
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
-    "imagemagick": "7.1.2.31-1",
+    "imagemagick": "7.1.2.32-1",
     "imath": "3.2.3-1",
     "imv": "5.0.1-2",
-    "iniparser": "4.2.6-2",
-    "inotify-tools": "4.25.9.0-1",
+    "iniparser": "4.3.2-1",
+    "inotify-tools": "4.26.262-1",
     "iproute2": "7.2.0-1",
     "iptables": "1:1.8.13-1",
     "iputils": "20250605-1",
@@ -185,7 +184,7 @@
     "jack2": "1.9.22-2",
     "jansson": "2.15.1-1",
     "jbigkit": "2.1-8",
-    "jemalloc": "1:5.3.1-3",
+    "jemalloc": "1:5.4.0-1",
     "jq": "1.8.2-1",
     "json-c": "0.19-1",
     "json-glib": "1.10.8-1",
@@ -196,10 +195,10 @@
     "l-smash": "2.14.5-4",
     "lame": "4.0-1",
     "lcms2": "2.19.1-1",
-    "leancrypto": "1.9.0-1",
+    "leancrypto": "1.9.1-1",
     "leptonica": "1.87.0-2",
-    "less": "1:704-1",
-    "libadwaita": "1:1.9.3-1",
+    "less": "1:710-1",
+    "libadwaita": "1:1.9.4-1",
     "libaec": "1.1.7-1",
     "libarchive": "3.8.9-1",
     "libasan": "16.2.1+r23+gd564253eb6c8-1",
@@ -226,7 +225,7 @@
     "libbsd": "0.12.2-2",
     "libbytesize": "2.12-3",
     "libcap": "2.78-1",
-    "libcap-ng": "0.9.5-1",
+    "libcap-ng": "0.9.6-1",
     "libcdio": "2.4.0-1",
     "libcdio-paranoia": "10.2+2.0.2-3",
     "libcgif": "0.5.4-1",
@@ -236,7 +235,7 @@
     "libcups": "2:2.4.19-1",
     "libdaemon": "0.14-6",
     "libdatrie": "0.2.14-1",
-    "libde265": "1.1.2-1",
+    "libde265": "1.1.3-1",
     "libdecor": "0.2.5-1",
     "libdeflate": "1.26-1",
     "libdisplay-info": "0.3.0-1",
@@ -244,7 +243,7 @@
     "libdrm": "2.4.134-1",
     "libdvdnav": "7.0.0-1",
     "libdvdread": "7.1.1-1",
-    "libdwarf": "1:2.3.2-1",
+    "libdwarf": "1:2.3.3-1",
     "libebur128": "1.2.6-2",
     "libedit": "20260512_3.1-1",
     "libei": "1.6.0-1",
@@ -259,17 +258,17 @@
     "libfreeaptx": "0.2.2-1",
     "libfyaml": "0.9.6-3",
     "libgcc": "16.2.1+r23+gd564253eb6c8-1",
-    "libgcrypt": "1.12.3-1",
+    "libgcrypt": "1.12.4-1",
     "libgfortran": "16.2.1+r23+gd564253eb6c8-1",
     "libgirepository": "1.86.0-2",
     "libgit2": "1:1.9.7-1",
     "libglvnd": "1.7.0-3",
     "libgomp": "16.2.1+r23+gd564253eb6c8-1",
     "libgpg-error": "1.61-1",
-    "libgsf": "1.14.58-1",
+    "libgsf": "1.14.60-1",
     "libgudev": "238-3",
     "libgxps": "0.3.2-5",
-    "libheif": "1.23.4-1",
+    "libheif": "1.23.5-1",
     "libhwasan": "16.2.1+r23+gd564253eb6c8-1",
     "libice": "1.1.2-1",
     "libidn2": "2.3.8-1",
@@ -278,7 +277,7 @@
     "libimobiledevice": "1.4.0-2",
     "libimobiledevice-glue": "1.3.2-1",
     "libinih": "62-2",
-    "libinput": "1.31.3-1",
+    "libinput": "1.32.0-1",
     "libiptcdata": "1.0.5-5",
     "libisl": "0.28-1",
     "libjpeg-turbo": "3.2.0-2",
@@ -286,13 +285,13 @@
     "libksba": "1.8.1-1",
     "liblc3": "1.1.3-2",
     "libldac": "2.0.2.6-1",
-    "libldap": "2.6.13-1",
+    "libldap": "2.7.1-1",
     "liblqr": "0.4.3-1",
     "liblsan": "16.2.1+r23+gd564253eb6c8-1",
     "libluv": "1.52.1-1",
     "libmakepkg-dropins": "20-2",
     "libmatio": "1.5.30-1",
-    "libmd": "1.2.0-1",
+    "libmd": "1.3.0-1",
     "libmm-glib": "1.24.2-1",
     "libmnl": "1.0.5-2",
     "libmodplug": "0.8.9.0-7",
@@ -309,7 +308,7 @@
     "libngtcp2": "1.25.0-1",
     "libnice": "0.1.24-1",
     "libnl": "3.12.0-1",
-    "libnm": "1.58.1-1",
+    "libnm": "1.58.1-2",
     "libnotify": "0.8.8-1",
     "libnsbmp": "0.1.7-2",
     "libnsgif": "1.0.0-2",
@@ -320,16 +319,16 @@
     "libopenmpt": "0.8.9-1",
     "libosinfo": "1.12.0-3",
     "libp11-kit": "0.26.5-1",
-    "libpcap": "1.10.7-1",
+    "libpcap": "1.11.0-1",
     "libpciaccess": "0.19-1",
     "libpgm": "5.3.128-4",
     "libpipeline": "1.5.8-1",
-    "libpipewire": "1:1.6.8-1",
-    "libplacebo": "7.360.1-6",
+    "libpipewire": "1:1.6.9-1",
+    "libplacebo": "7.360.1-7",
     "libplist": "2.7.0-3",
-    "libpng": "1.6.58-2",
-    "libportal": "0.10.0-1",
-    "libportal-gtk4": "0.10.0-1",
+    "libpng": "1.6.59-1",
+    "libportal": "0.11.0-1",
+    "libportal-gtk4": "0.11.0-1",
     "libproxy": "0.5.12-1",
     "libpsl": "0.21.5-2",
     "libpulse": "17.0+r98+gb096704c0-1",
@@ -337,11 +336,11 @@
     "libraqm": "0.11.0-1",
     "libraw": "0.22.2-1",
     "libraw1394": "2.1.2-4",
-    "librsvg": "2:2.62.3-1",
+    "librsvg": "2:2.62.4-1",
     "libsamplerate": "0.2.2-3",
     "libsasl": "2.1.28-5",
     "libseccomp": "2.6.0-1",
-    "libsecret": "0.21.7-1",
+    "libsecret": "0.21.8.2-1",
     "libsixel": "1.10.5-1",
     "libsm": "1.2.6-1",
     "libsndfile": "1.2.2-4",
@@ -359,7 +358,7 @@
     "libthai": "0.1.30-1",
     "libtheora": "1.2.0-1",
     "libtiff": "4.7.2-1",
-    "libtirpc": "1.3.7-1",
+    "libtirpc": "1.3.8-1",
     "libtool": "2.6.2-5",
     "libtsan": "16.2.1+r23+gd564253eb6c8-1",
     "libubsan": "16.2.1+r23+gd564253eb6c8-1",
@@ -370,19 +369,19 @@
     "libusb": "1.0.30-1",
     "libusbmuxd": "2.1.1-2",
     "libutempter": "1.2.3-1",
-    "libutf8proc": "2.11.3-1",
+    "libutf8proc": "2.12.0-1",
     "libuv": "1.52.1-2",
     "libva": "2.24.1-1",
     "libvdpau": "1.5-4",
     "libverto": "0.3.2-6",
-    "libvips": "8.18.6-1",
+    "libvips": "8.18.7-1",
     "libvorbis": "1.3.7-4",
     "libvpl": "2.17.0-1",
     "libvpx": "1.17.0-1",
     "libvterm": "0.3.3-2",
     "libwacom": "2.20.0-1",
     "libwebp": "1.6.0-2",
-    "libwireplumber": "0.5.17-1",
+    "libwireplumber": "0.5.18-1",
     "libx11": "1.8.13-1",
     "libxau": "1.0.12-1",
     "libxcb": "1.17.0-1",
@@ -418,10 +417,10 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.4.arch1-2",
+    "linux": "7.2.8.arch1-2",
     "linux-api-headers": "7.2-1",
     "llhttp": "9.3.1-1",
-    "llvm-libs": "22.1.8-2",
+    "llvm-libs": "23.1.1-1",
     "lm_sensors": "1:3.6.2-1",
     "lmdb": "0.9.35-1",
     "localsearch": "3.11.2-1",
@@ -435,11 +434,11 @@
     "m4": "1.4.21-2",
     "make": "4.4.1-3",
     "man-db": "2.13.1-2",
-    "md4c": "0.5.3-1",
+    "md4c": "0.6.0-2",
     "mdadm": "4.6-2",
-    "mesa": "1:26.2.2-1",
+    "mesa": "1:26.2.4-1",
     "minizip": "1:1.3.2-3",
-    "mkinitcpio": "41.1-1",
+    "mkinitcpio": "42.2-1",
     "mkinitcpio-busybox": "1.36.1-1",
     "mobile-broadband-provider-info": "20251101-1",
     "mpdecimal": "4.0.1-3",
@@ -448,20 +447,21 @@
     "mpv": "1:0.41.0-6",
     "msgpack-c": "7.0.1-1",
     "mtdev": "1.1.7-1",
-    "mujs": "1.3.9-1",
+    "mujs": "1.3.10-1",
     "muparser": "2.3.5-2",
     "nautilus": "50.3.1-1",
     "ncurses": "6.6-2",
     "neovim": "0.12.5-1",
     "nettle": "4.0-1",
-    "networkmanager": "1.58.1-1",
+    "networkmanager": "1.58.1-2",
     "nftables": "1:1.1.7-3",
-    "noto-fonts": "1:2026.09.01-1",
+    "noto-fonts": "1:2026.10.01-1",
     "noto-fonts-cjk": "20240730-1",
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
     "nspr": "4.40-1",
-    "nss": "3.128-1",
+    "nss": "3.130-1",
+    "numactl": "2.0.19-1",
     "ocl-icd": "2.3.5-1",
     "omacalc": "0.2.2-1",
     "omacut": "0.4.0-1",
@@ -472,37 +472,37 @@
     "oniguruma": "6.9.10-1",
     "openal": "1.25.2-2",
     "opencore-amr": "0.1.6-2",
-    "openexr": "3.4.15-1",
+    "openexr": "3.5.2-1",
     "openh264": "2.6.0-2",
     "openjpeg2": "2.5.4-1",
-    "openjph": "0.31.0-1",
+    "openjph": "0.32.0-1",
     "openssh": "10.5p1-1",
-    "openssl": "3.6.4-1",
+    "openssl": "3.6.5-1",
     "opus": "1.6.1-1",
-    "orc": "0.4.43-1",
+    "orc": "0.4.44-1",
     "osinfo-db": "20260812-1",
     "p11-kit": "0.26.5-1",
     "pacman": "7.1.0.r9.g54d9411-2",
     "pacman-contrib": "1.13.1-1",
     "pacman-mirrorlist": "20260610-1",
-    "pam": "1.7.2-2",
+    "pam": "1.7.3-1",
     "pambase": "20260616-1",
     "pango": "1:1.58.2-1",
-    "parted": "3.7-1",
+    "parted": "3.8-1",
     "patch": "2.8-1",
     "pciutils": "3.15.0-1",
     "pcre": "8.45-5",
-    "pcre2": "10.48-1",
-    "pcsclite": "2.5.1-1",
+    "pcre2": "10.49-1",
+    "pcsclite": "2.5.2-1",
     "perl": "5.42.2-2",
     "perl-error": "0.17030-3",
     "perl-mailtools": "2.22-3",
     "perl-timedate": "2.35-1",
     "pinentry": "1.3.3-1",
-    "pipewire": "1:1.6.8-1",
-    "pipewire-alsa": "1:1.6.8-1",
-    "pipewire-audio": "1:1.6.8-1",
-    "pipewire-pulse": "1:1.6.8-1",
+    "pipewire": "1:1.6.9-1",
+    "pipewire-alsa": "1:1.6.9-1",
+    "pipewire-audio": "1:1.6.9-1",
+    "pipewire-pulse": "1:1.6.9-1",
     "pixman": "0.46.4-1",
     "pkgconf": "3.0.7-1",
     "plocate": "1.1.25-1",
@@ -515,7 +515,7 @@
     "psmisc": "23.7-2",
     "pugixml": "1.16-1",
     "python": "3.14.7-1",
-    "python-dbus": "1.4.0-2",
+    "python-dbus": "1.5.0-1",
     "python-docopt": "0.6.2-15",
     "python-gobject": "3.56.3-1",
     "python-pyxdg": "0.28-7",
@@ -523,7 +523,7 @@
     "qoi": "r350.4461cc3-1",
     "qrencode": "4.1.1-4",
     "qt6-base": "6.11.2-3",
-    "qt6-declarative": "6.11.2-1",
+    "qt6-declarative": "6.11.2-2",
     "qt6-multimedia": "6.11.2-1",
     "qt6-multimedia-ffmpeg": "6.11.2-1",
     "qt6-svg": "6.11.2-1",
@@ -532,18 +532,18 @@
     "quickshell": "0.3.1-1",
     "rav1e": "0.8.1-3",
     "re2": "2:2025.11.05-6",
-    "readline": "8.3.003-1",
+    "readline": "8.3.6-1",
     "ripgrep": "15.2.0-1",
     "rubberband": "4.0.0-2",
     "sbc": "2.2-1",
     "sdbus-cpp": "2.3.1-1",
     "sddm": "0.21.0-7",
-    "sdl2-compat": "2.32.72-1",
-    "sdl3": "3.4.16-1",
+    "sdl2-compat": "2.32.74-1",
+    "sdl3": "3.4.18-1",
     "seatd": "0.9.3-1",
     "sed": "4.10-2",
     "serd": "0.32.10-1",
-    "shaderc": "2026.3-1",
+    "shaderc": "2026.4-1",
     "shadow": "4.20.0.arch1-1",
     "shared-mime-info": "2.5.1-2",
     "slang": "2.3.3-4",
@@ -556,17 +556,17 @@
     "spandsp": "0.0.6-7",
     "speex": "1.2.1-2",
     "speexdsp": "1.2.1-2",
-    "spirv-tools": "1:1.4.357.0-1",
+    "spirv-tools": "1:1.4.363.0-1",
     "sqlite": "3.53.4-1",
     "sratom": "0.6.22-1",
     "srt": "1.5.7-1",
     "starship": "1.26.0-1",
     "sudo": "1.9.17.p2-6",
     "svt-av1": "4.2.0-1",
-    "systemd": "261.2-1",
-    "systemd-libs": "261.2-1",
-    "systemd-resolvconf": "261.2-1",
-    "systemd-sysvcompat": "261.2-1",
+    "systemd": "262-1",
+    "systemd-libs": "262-1",
+    "systemd-resolvconf": "262-1",
+    "systemd-sysvcompat": "262-1",
     "tar": "1.35-5",
     "tensaku": "0.29.0-1",
     "tesseract": "5.5.3-1",
@@ -589,30 +589,30 @@
     "ttf-jetbrains-mono-nerd": "3.5.1-2",
     "ttf-liberation": "2.1.5-2",
     "ttfx": "0.3.2-1",
-    "tzdata": "2026c-1",
+    "tzdata": "2026e-1",
     "uchardet": "0.0.8-4",
     "udiskie": "2.7.0-2",
     "udisks2": "2.11.2-1",
     "unibilium": "2.1.4-1",
     "unzip": "6.0-23",
-    "upower": "1.91.3-1",
-    "util-linux": "2.42.3-1",
-    "util-linux-libs": "2.42.3-1",
-    "uwsm": "0.26.7-1",
+    "upower": "1.91.5-1",
+    "util-linux": "2.42.4-1",
+    "util-linux-libs": "2.42.4-1",
+    "uwsm": "0.27.0-1",
     "v4l-utils": "1.32.0-2",
-    "vapoursynth": "79-1",
+    "vapoursynth": "80-1",
     "vid.stab": "1.1.2-1",
-    "vmaf": "3.2.0-1",
+    "vmaf": "3.2.1-1",
     "volume_key": "0.3.12-12",
-    "vulkan-icd-loader": "1.4.357.0-1",
-    "vulkan-mesa-implicit-layers": "1:26.2.2-1",
-    "vulkan-swrast": "1:26.2.2-1",
-    "vulkan-virtio": "1:26.2.2-1",
+    "vulkan-icd-loader": "1.4.363.0-1",
+    "vulkan-mesa-implicit-layers": "1:26.2.4-1",
+    "vulkan-swrast": "1:26.2.4-1",
+    "vulkan-virtio": "1:26.2.4-1",
     "wayland": "1.26.0-1",
     "wayland-protocols": "1.49-1",
     "webrtc-audio-processing-1": "1.3-5",
     "which": "2.25-1",
-    "wireplumber": "0.5.17-1",
+    "wireplumber": "0.5.18-1",
     "wl-clipboard": "1:2.3.0-1",
     "woff2-font-awesome": "7.3.1-1",
     "wpa_supplicant": "2:2.12-1",
@@ -631,7 +631,7 @@
     "xdg-desktop-portal": "1.22.1-2",
     "xdg-desktop-portal-gtk": "1.15.3-1",
     "xdg-desktop-portal-hyprland": "1.4.1-2",
-    "xdg-terminal-exec": "0.14.3-1",
+    "xdg-terminal-exec": "0.14.3-2",
     "xdg-user-dirs": "0.20-1",
     "xdg-user-dirs-gtk": "0.16-1",
     "xdg-utils": "1.2.1-2",
@@ -647,11 +647,11 @@
     "xorg-xwayland": "24.1.13-1",
     "xorgproto": "2025.1-1",
     "xvidcore": "1.3.7-4",
-    "xxhash": "0.8.3-1",
-    "xz": "5.8.3-1",
+    "xxhash": "0.8.4-1",
+    "xz": "5.8.4-1",
     "yay": "13.0.1-1",
     "yoga": "3.2.1-1",
-    "yyjson": "0.12.0-1",
+    "yyjson": "0.13.0-1",
     "zbar": "0.23.93-7",
     "zeromq": "4.3.5-3",
     "zimg": "3.0.6-2",
-- 
2.55.0

